src/qt/guiutil.cpp

```diff
@@ -567,7 +567,10 @@ TableViewLastColumnResizingFixer::TableViewLastColumnResizingFixer(QTableView* t
 #ifdef WIN32
 boost::filesystem::path static StartupShortcutPath()
 {
-    return GetSpecialFolderPath(CSIDL_STARTUP) / "Bitcoin.lnk";
```

Comment:
Why add a split for regtest here when bitcoin doesnt? I dont think we want to add things that really should just be upstreamed and then tweaked in elements?